examples/js/snippet.js

```diff
@@ -22,7 +22,8 @@
     __productcat_obj.methods = [
       'set',
```

Comment:
what is the "set" method for?

Reply:
this is to set global parameters. like app_key and cid. and customized endpoint. i will add customized endpoint here. ok ?

Reply:
ok